app/src/common/shared/org/mozilla/vrbrowser/ui/NavigationURLBar.java

```diff
@@ -135,6 +135,8 @@ else if (aURL.startsWith("resource:") || SessionStore.get().isHomeUri(aURL))
                 aURL = "";
             else if (aURL.startsWith("data:") && SessionStore.get().isCurrentSessionPrivate())
                 aURL = "";
+            else if (aURL.startsWith("javascript:"))
```

Comment:
nah, we should not always hide them. if a user enters a URL like this or gets redirected/clicked, it should show in the URL bar. can we hide it for only when the Home page is loaded?

Reply:
Fixed